docs/LEARNING.md

```diff
@@ -1,5 +1,30 @@
 # Learning
 
+COBOL has a variety of resources to help both experiences programmers and beginners, though some might be obscure. Luckily, here's a compilation of some popular sources.
+
+## Books
+
+[Beginning COBOL for Programmers][Coughlan Book]  
```

Comment:
We usually use bullet point in this document, which makes it slightly easier for readers to scan.

```suggestion
- [Beginning COBOL for Programmers][Coughlan Book]  
```